.claude/commands/steering-update.md

```
@@ -0,0 +1,100 @@
+- Recent commits: !`git log --oneline -15`
+- Modified files (last 10 commits): !`git diff --name-only HEAD~10`
+- Added files: !`git diff --name-status HEAD~10 | grep "^A"`
+- Deleted files: !`git diff --name-status HEAD~10 | grep "^D"`
+- Renamed files: !`git diff --name-status HEAD~10 | grep "^R"`
+
+### Dependency Changes
+- Package.json changes: !`git diff HEAD~10 -- package.json 2>/dev/null || echo "No package.json changes"`
+- Requirements.txt changes: !`git diff HEAD~10 -- requirements.txt 2>/dev/null || echo "No requirements.txt changes"`
+- Config changes: !`git diff HEAD~10 --name-only | grep -E "(tsconfig|webpack|vite|rollup|babel|eslint|prettier)"`
+
+### Current Project State
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)